app/assets/stylesheets/govuk-component/_option-select.scss

```diff
@@ -89,6 +89,7 @@
 
         @include device-pixel-ratio() {
           background-size: 32px 50px;
+          background: image-url('govuk-component/option-select-toggle-sprite.png') no-repeat right 5px;
```

Comment:
As far as I am aware using the `background` shorthand sets the `background-size` to the default `auto` ([from the spec](http://www.w3.org/TR/css3-background/#the-background)). So this new line will cancel out the `background-size: 32px 50px;` and actually makes the whole `device-pixel-ratio` block redundant. Could we instead just remove the block?